client/ayon_hiero/api/lib.py

```diff
@@ -415,36 +415,41 @@ def get_track_openpype_tag(track):
         return None
     for tag in _tags:
         # return only correct tag defined by global name
-        if OPENPYPE_TAG_NAME in tag.name():
+        if AYON_TAG_NAME in tag.name():
             return tag
 
 
-def get_track_openpype_data(track, container_name=None):
+def get_track_ayon_data(track, container_name=None):
     """
-    Get track's openpype tag data.
+    Get track's AYON tag data.
 
     Attributes:
         trackItem (hiero.core.VideoTrack): hiero object
 
     Returns:
-        dict: data found on pype tag
+        dict: data found on the AYON tag
     """
     return_data = {}
     # get pype data tag from track item
-    tag = get_track_openpype_tag(track)
+    tag = get_track_ayon_tag(track)
 
     if not tag:
         return None
 
     # get tag metadata attribute
     tag_data = deepcopy(dict(tag.metadata()))
+    if tag_data.get("tag.json_metadata"):
+        tag_data = json.loads(tag_data["tag.json_metadata"])
 
     for obj_name, obj_data in tag_data.items():
         obj_name = obj_name.replace("tag.", "")
 
         if obj_name in ["applieswhole", "note", "label"]:
             continue
```

Comment:
This may be slightly faster - not recreating the list each time and making the lookup o1 (even though the list is super small anyway so hardly relevant)
```suggestion
    ignore_names  = {"applieswhole", "note", "label"}
    for obj_name, obj_data in tag_data.items():
        obj_name = obj_name.replace("tag.", "")

        if obj_name in ignore_names:
            continue
```